src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const todoList = {
+  id: "First",
+  title: "Review notes",
+  id: "Second",
+  title: "Listen to next 2 React sections",
+  id: "Third",
+  title: "Make dinner",
+}
```

Comment:
Let's reformat this to be an **array** where each element of the array is an _object_. Then, each object should be a Todo, containing it's own `id` and `title` key.

Hint: 

```js
[{},{},{}]
```